iterative/utils/helpers.go

```diff
@@ -7,6 +7,29 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 )
 
+var (
+	SynthRegions = map[string]map[string]string{
+		"aws": {
+			"us-east":  "us-east-1",
+			"us-west":  "us-west-1",
+			"eu-north": "eu-north-1",
+			"eu-west":  "eu-west-1",
+		},
+		"gcp": {
+			"us-east":  "us-east1-c",
+			"us-west":  "us-west1-b",
+			"eu-north": "europe-north1-a",
+			"eu-west":  "europe-west1-d",
+		},
+		"az": {
```

Comment:
@dacbd cloud in `machine` and `runner` is `azure` not `az`

@0x2b3bfa0 we should have maintained the consistency 

I do not feel confident to release this without having tested every cloud. 
Please @dacbd try to fix it just only for AWS vendor without reaching a wider scope

Reply:
I mean roll back utils.Get region

Reply:
> _@0x2b3bfa0 we should have maintained the consistency_

We do, to some extent.

https://github.com/iterative/terraform-provider-iterative/blob/01af1440a6232c23d46fdd1967bb6d452783cd1d/iterative/resource_machine.go#L200

Reply:
> > _@0x2b3bfa0 we should have maintained the consistency_
> 
> We do, to some extent.
> 
> https://github.com/iterative/terraform-provider-iterative/blob/01af1440a6232c23d46fdd1967bb6d452783cd1d/iterative/resource_machine.go#L200

We should support then `az` in the `machine` and `runner`

Reply:
We _do_ support `az` in both the `machine` and `runner` resources. At least, before merging this pull request.

Reply:
One should be an alias of the other to avoid issues like this one

Reply:
Do you mean replacing the value intead of accepting both?

Reply:
Yes, an alias